ci/test/e2e.go

```diff
@@ -118,16 +141,29 @@ func (r *runner) setup() error {
 	if err := r.compose("build"); err != nil {
```

Comment:
This should probably go to the other setup method where we start app containers.